keg_auth/model/__init__.py

```diff
@@ -75,7 +76,24 @@ class UserMixin(object):
     # is_active defines the complexities of how to determine what users are active. For instance,
     #   if email is in scope, we need to have an additional flag to verify users, and that would
     #   get included in is_active logic.
-    is_active = is_enabled
+    @hybrid_property
+    def is_active(self):
+        is_disabled = self.disable_date is not None and self.disable_date < arrow.utcnow()
+        return not is_disabled and self.is_enabled
+
+    @is_active.expression
+    def is_active(cls):
+        not_disabled = sa_sql.or_(
```

Comment:
When this gets moved to the separate hybrid property, let's make the operators and arrangement of logical expressions consistent with the instance version. What's here is logically equivalent, but rendering an actual equivalence would promote readability.